Fabric.Identity.API/scripts/Register-Identity-IdPSS.ps1

```diff
@@ -0,0 +1,47 @@
+param(
+    [ValidateScript({
+        if (!(Test-Path $_)) {
+            throw "Path $_ does not exist. Please enter valid path to the install.config."
+        }
+        if (!(Test-Path $_ -PathType Leaf)) {
+            throw "Path $_ is not a file. Please enter a valid path to the install.config."
+        }
+        return $true
+    })] 
+    [string] $installConfigPath = "$PSScriptRoot\install.config",
+    [Parameter(Mandatory=$true)]
+    [string[]] $tenants,
```

Comment:
I think these should become install.config params, for the reasons stated above.